<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpOverview</class>
 <widget class="QWidget" name="JumpOverview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="10,1,10,1">
   <item row="0" column="0" rowspan="5">
    <widget class="QListWidget" name="items"/>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLabel" name="type"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="buddy">
      <cstring>name</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="name">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="bDelete">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Text:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="KSqueezedTextLabel" name="textContent"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="pageLabel">
     <property name="text">
      <string>Page:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="page"/>
   </item>
   <item row="4" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header>ksqueezedtextlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
